src/components/routers/GroupRouter.test.tsx

```diff
@@ -61,9 +57,9 @@ describe('GroupRouter >', () => {
         describe('toCases >', () => {
             it('should map a list of group SearchResults to a list of CaseElementProps', () => {
                 const groups = extractGroups(node);
-                toCases(groups).forEach((cäse, idx) => {
-                    expect(cäse.kase.uuid).toBe(groups[idx].id);
-                    expect(cäse.exitName).toBe(groups[idx].name);
+                toCases(groups).forEach((groupCase, idx) => {
```

Comment:
`groupCase` over `cäse` (was just so tired of writing `kase`, sucks `case` is reserved)

Reply:
this is what happens when you write a programming language with a programming language

Reply:
😆